inspirehep/modules/records/receivers.py

```diff
@@ -334,3 +335,15 @@ def add_recids_and_validate(sender, json, *args, **kwargs):
     """Ensure that recids are generated before being validated."""
     populate_recid_from_ref(sender, json, *args, **kwargs)
     references_validator(sender, json, *args, **kwargs)
+
+
+@before_record_update.connect
+def check_if_record_is_going_to_be_deleted(sender, *args, **kwargs):
```

Comment:
Signature should be `def check_if_record_is_going_to_be_deleted(sender, json, *args, **kwargs):`

It doesn't matter right now, but I can guarantee that people are going to start copying the wrong signature, and then it will matter.